### ISSUE TITLE
Working on Mac!! Only can’t get rid of free trial

### ISSUE THREAD

**Issue by jaysid100** [sunenielsen/tribaltrouble#29]
Hello! We’ve gotten the file to open (after it gave “damaged” error) by following the instructions from this YouTube video to do some work in Terminal: https://m.youtube.com/watch?v=DgEA6qu4kYc

Only issue now is that the free trial isn’t going away. The registration file is in the correct folder as far as I can tell! Could someone help me troubleshoot the registration??

**Comment by veslav3** [comment]
@jaysid100 can you provide details on your Mac version? And also the folders.

**Comment by jaysid100** [comment]
Absolutely! Give me a little bit and I'll provide that info and screenshots of file locations @veslav3

**Comment by jaysid100** [comment]
Hey @veslav3 ! Here's the information in screenshots. It's also valuable to mention that I had to follow the contents of this YouTube video to get my Mac to recognize TT as not "broken," which included some work in the Terminal: https://m.youtube.com/watch?v=DgEA6qu4kYc

I'm available at any point to answer additional questions as well. Thank you for your help- my wife will be ecstatic if we access the full version, this was a childhood game for her. 

![image](https://github.com/user-attachments/assets/c01fcd1d-656f-4161-97b0-f03117bbe945)

![image](https://github.com/user-attachments/assets/ec5eede7-7a71-4346-99ac-e03d4431e1db)

![image](https://github.com/user-attachments/assets/845112fd-8bfd-4d5c-b314-4e051f36bb9d)
